Techome/Techome/Views/CaffeineBook/CaffeineBookView.swift

```diff
@@ -0,0 +1,228 @@
+//
+//  CaffeineBookView.swift
+//  Techome
+//
+//  Created by Yeongwoo Kim on 2022/06/14.
+//
+
+import SwiftUI
+
+struct CaffeineBookLayoutValue {
+    struct Paddings {
+        static let topBarCategoryHorizontal: CGFloat = 0
+        static let topBarCategoryTop: CGFloat = 35
+        static let topBarCategoryBottom: CGFloat = 10
```

Comment:
```suggestion
        enum TopBar {
            enum Category {
                static let horizontal: CGFloat = 0
                static let top: CGFloat = 35
                static let bottom: CGFloat = 10
            }
        }
```

```swift
CaffeineBookLayoutValue. Paddings.TopBar.Category.top
```

enum을 사용해서 공통되는 category를 구분하면 꺼내쓸 때도 자동완성으로 찾기 쉽답니다 ~

Reply:
팁 감사합니다 수정하겠습니다!